index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <section class="toolbar">
+            <a id="logo" href="#">My Logo</a>
+            <div class="nav">
+                <img src="./assets/images/menu.png">
+            </div>
+        </section>
+        <section class="headline">
+            <h1>I’m Alexandra. Glad to see you!</h1>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my 
+                work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <a class="connect" href="#">LET’S CONNECT</a>
+            <a class="social"><img src="./assets/images/social-3.png"></a>
+            <a class="social"><img src="./assets/images/social-4.png"></a>
+            <a class="social"><img src="./assets/images/social-3.png"></a>
+            <a class="social"><img src="./assets/images/social-1.png"></a>
+            <a class="social"><img src="./assets/images/social-2.png"></a>
```

Comment:
- [ ] Since this is a list of social media icons, it would be nice to use a `ul` tag and wrap each icon with a `li` and `a` tags for semantics reason.